targets/psa_switch/pswitch_CLI.py

```diff
@@ -101,6 +123,34 @@ def get_json_key(key):
             elif name == "rate_count":
                 meter_array.rate_count = value
 
+    for j_random in get_json_key("extern_instances"):
+        if j_random["type"] != "Random":
+            continue
+        psa_random = PsaRandom(j_random["name"], j_random["id"])
+        attribute_values = j_meter.get("attribute_values", [])
+        for attr in attribute_values:
+            name = attr.get("name", [])
+            val_type = attr.get("type", [])
+            value = attr.get("value", [])
+            if name == "min":
+                psa_random.min = value
+            elif name == "max":
+                psa_random.max = value
+
+    suffix_count = runtime_CLI.Counter()
+    for res_type, res_dict in [
+            (ResType.random, RANDOMS)]:
+        for name, res in res_dict.items():
+            suffix = None
+            for s in reversed(name.split('.')):
+                suffix = s if suffix is None else s + '.' + suffix
+                key = (res_type, suffix)
+                runtime_CLI.SUFFIX_LOOKUP_MAP[key] = res
+                suffix_count[key] += 1
+    for key, c in suffix_count.items():
+        if c > 1:
+            del runtime_CLI.SUFFIX_LOOKUP_MAP[key]
```

Comment:
Had to duplicate this piece of code (line 140-150) from [tools/runtime_CLI.py line 380](https://github.com/p4lang/behavioral-model/blob/master/tools/runtime_CLI.py#L380) as I needed to add `RANDOMS` to `SUFFIX_LOOKUP_MAP`.

I could put `RANDOMS` in `tools/runtime_CLI.py` add avoid this duplication, however, I did not because I do not want to put `RANDOMS` (psa-random specific code) to `tools/runtime_CLI.py` which not architecture specific.

Comments on whether this code organizing decision make sense and suggestions are appreciated!

Reply:
can we write a function in `tools/runtime_CLI.py` which does that computation for a provided list of `(type, dict)` tuples and use that function in both places?

Reply:
more importantly, why are you modifying pswitch_CLI.py here? which runtime functions are you planning to support?

Reply:
Haha, you are right, there is no runtime function I am trying to support and I do not need to modify pswitch_CLI.py at all. I will remove this chunk of code.

Reply:
Haha, you are right, there is no runtime function I am trying to support and I do not need to modify pswitch_CLI.py at all. I will remove this chunk of code.